components/21-atoms/link/_link.scss

```diff
@@ -138,11 +138,13 @@ a {
   ///
   &[href^="http://"]:not(.no-icon):not(.tag),
   &[href^="https://"]:not(.no-icon):not(.tag) {
-    @include icon(arrow-right-top, after);
-    margin-right: 1.6rem;
+    &:not([href*='stad.gent']) {
```

Comment:
Stad.gent specifieke code hoort niet thuis in de stijlgids. Deze moet verhuizen naar het stad.gent project.